curriculum/challenges/english/06-quality-assurance/advanced-node-and-express/use-a-template-engines-powers.md

```diff
@@ -12,15 +12,37 @@ One of the greatest features of using a template engine is being able to pass va
 
 In your Pug file, you're able to use a variable by referencing the variable name as `#{variable_name}` inline with other text on an element or by using an equal sign on the element without a space such as `p=variable_name` which assigns the variable's value to the p element's text.
 
-We strongly recommend looking at the syntax and structure of Pug [here](https://github.com/pugjs/pug) on GitHub's README. Pug is all about using whitespace and tabs to show nested elements and cutting down on the amount of code needed to make a beautiful site.
+ Pug is all about using whitespace and tabs to show nested elements and cutting down on the amount of code needed to make a beautiful site. Here is an Example:
+ 
```

Comment:
We might still suggest the camper read the Pug documentation without linking to it?

```suggestion
 Pug is all about using whitespace and tabs to show nested elements and cutting down on the amount of code needed to make a beautiful site. Read the Pug documentation for more information on usage and syntax.
 
 Here is an example: 
```

Reply:
I do think that the example is insufficient, but people will find inconvenient to be asked to check something without a link to it. I think the best action is to hold this issue too untill an article about pug is written

Reply:
It's really just meant as a small reminder to RTFM. Which people surprisingly often do not.

Reply:
You know what, that is funny. I am in